main_chain.js

```diff
@@ -420,6 +420,38 @@ function updateMainChain(conn, from_unit, last_added_unit, onDone){
 		});
 	}
 
+	function findMinMcWitnessedLevel(tip_unit, min_level, arrWitnesses, handleMinMcWl){
+		var arrCollectedWitnesses = [];
+		var min_mc_wl = Math.pow(2, 31) - 1;
```

Comment:
Infinity would be more future-proof

Reply:
I'll change it to Number.POSTIVE_INFINITY